services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_specs.py

```diff
@@ -125,6 +125,8 @@ def _update_resource_limits_and_reservations(
     # example: '2.3' -> 2 ; '3.7' -> 3
     docker_compose_major_version: int = int(service_spec["version"].split(".")[0])
     for spec_service_key, spec in service_spec["services"].items():
+        if spec_service_key not in service_resources:
+            continue
```

Comment:
so services w/o resources specified, they will not include resource section in the compose specs, right?

Reply:
This is correct. This is the case of the proxies we inject in the networks when running with restricted internet access.

Reply:
And now that I look at the code, this error happened, because of a retry in the the `docker-compose up`.

Reply:
since you fixed the issue. does this change really need to stay? normally we do not want services without resources defined